IdParser/Parsers/Id/EyeColorParser.cs

```diff
@@ -38,6 +38,11 @@ public override void ParseAndSet(string input)
             {
                 IdCard.EyeColor = EyeColor.Brown;
             }
+            // West Verginia doesn't follow the abbreviation scheme for brown
```

Comment:
Can you fix the Virginia typo? Also, it's probably a good idea to add this same BN logic to the `HairColorParser`.